README.md

```diff
@@ -92,6 +92,7 @@
 | [RISCV_GNU](https://github.com/riscv/riscv-gnu-toolchain) |  -   |  -  |  -  |    -    | riscv32-unknown-linux-gnu-gcc | riscv64-unknown-linux-gnu-gcc |
 | [RISCV64_GCC](https://packages.ubuntu.com/bionic/gcc-riscv64-linux-gnu) |  -   |  -  |  -  |    -    |    -    | riscv64-linux-gnu-gcc |
 | [RISCV_XPACK](https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack) |  -   |  -  |  -  |    -    | riscv-none-elf-gcc | riscv-none-elf-gcc |
+| [RISCV_NONE](https://github.com/riscv/riscv-gnu-toolchain) |  -   |  -  |  -  |    -    | riscv64-elf-gcc | riscv64-elf-gcc |

```

Comment:
I think the URL should be https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/, right?

Reply:
I think it should be https://gcc.gnu.org/ as that is where the source is from

Reply:
Are you saying, we need build the compiler from the source?

If so, we need add those steps in build document.

All in all, I think we need a clear step-by-step document to tell people, how to get the compiler binary. (Either download/install it from somewhere, or build from source)

So far, I still don't understand.

Reply:
> Are you saying, we need build the compiler from the source?

No, I'm not saying that

> 
> If so, we need add those steps in build document.
> 
> All in all, I think we need a clear step-by-step document to tell people, how to get the compiler binary. (Either download/install it from somewhere, or build from source)

A user will need to install a GCC based RISC-V toolchain. By pointing to https://gcc.gnu.org/ we are saying you can use standard GCC instead of a fork (like https://github.com/riscv/riscv-gnu-toolchain which is a fork).

Most distros should expose a RISC-V toolchain, it should just involved installing it.

> 
> So far, I still don't understand.